src/containerapp/azext_containerapp/_models.py

```diff
@@ -131,6 +131,10 @@
     "latestRevision": False
 }
 
+StickySessions = {
```

Comment:
do we need to define this model explicitly here? Can't it be used like 'ipSecurityRestrictions' that does not need to be added separately here, we just have a mapping to it in the sdk_models.py.

Reply:
that was my reasoning as well but there is a `TrafficWeight` model in the same file `_models.py` as well hence i added StickySession struct

Reply:
ok, if that struct is being used in the custom.py file it makes sense to have it, else remove it. 
Ref: https://github.com/Azure/azure-cli-extensions/blob/81e5104f8aeb2267b5f3c5cac1e90fb0c45a05c2/src/containerapp/azext_containerapp/custom.py#L1767

Reply:
done